### ISSUE TITLE
Find something to do with excess power

### ISSUE THREAD

**Issue by CptWad** [d3athrow/vgstation13#8225]
![1455229699063](https://cloud.githubusercontent.com/assets/3959840/12993090/1445bf02-d0cd-11e5-87d8-c1e10a9d412b.png)
We could make portable SMES a cargo thing where cargo can buy empty batteries, engineering fills them then cargo can sell them for a profit.
least I think its a good idea

**Comment by CptWad** [comment]
also not me in the picture but I have had this idea for a while saw this and thought that a line had been crossed.

**Comment by 9600bauds** [comment]
1) Engineering makes quick'n'easy 2000MW TEG for muh moneys
2) Cargo doesn't give a shit like they always do / Engineering realizes money is worthless anyways seeing as the head 5K accounts always go untapped
3) Engineering makes 2000MW anyways, mostly in the form of autistic silicons waving around their newfound justification
4) 2000MW to grid every round

**Comment by CptWad** [comment]
yes but now engineers have new purpose to not just set up the engines and leave. Now they will be autisming their way to great new heights so they can beat the miners as most rich escapee.

**Comment by Shadowmech88** [comment]
The railgun in #8010 scales its damage with the power in the grid.
Though I'd hope 11GW doesn't become the new norm, since that'd mess with its balance.

**Comment by Jeroen52** [comment]
@Shadowmech88 First you need to reach 11GW, which is an achievement by itself.

**Comment by Shadowmech88** [comment]
@Jeroen52
1GW is the gun's current cap, at which point it can penetrate clear through half the station and gib anyone it hits.
This was based on the fact that I've never seen or heard of anyone hitting 1GW until now.

**Comment by Jeroen52** [comment]
@Shadowmech88 I have only hit about 80 MW, how much damage can that do?

**Comment by Shadowmech88** [comment]
@Jeroen52
That'd be 40 damage per shot.
I've included a graph of the relationship between the wattage and the gun's damage/penetration in #8010.

**Comment by AutismSpider** [comment]
one GW can be done just by dumping lot of cold gas into the burn room+cold loop of the TEG
![autism of the spider](https://cloud.githubusercontent.com/assets/10268342/12994615/a3b40800-d0e7-11e5-99e8-0fb5828893b7.PNG)

only mommis do this shit any way 
![mommi coming through](https://cloud.githubusercontent.com/assets/10268342/12994642/c0aa7a34-d0e7-11e5-8633-ff6432045f1f.png)

**Comment by Shadowmech88** [comment]
The TEG is truly the most casual easymode engine.

**Comment by Skullyton** [comment]
There was an idea to make an energy to matter condenser, but the math behind it was something along the lines of it taking a few hundred years to make on matter cartridge.

I'd be all for something like a better teleporter system, or bring the away missions portal back online and it need something like 250MW of its own in the grid to sustain.

**Comment by AutismSpider** [comment]
250MW+ can be done with just the super matter and 14 cold tanks of plasma good atmos tech/engi tech can do.

**Comment by uazuaz** [comment]
give objectives to all non-antag.
greentext autism will do the rest.

**Comment by TermedClepe** [comment]
Meteor shield - it would require loads of power to protect station from meteors, players would still hear dulled meteors impacts but in reality the meteors wouldn't appear if the shield is functioning properly.

**Comment by AutismSpider** [comment]
better then shield gens we have now they pop in one god damn hit form a meteor and take to long get back up

**Comment by 9600bauds** [comment]
That's actually a great idea, make autismal amounts of power give periodic compressed matter cartridges. Who gives a shit about realism.

**Comment by 9600bauds** [comment]
Also make wiring to grid illegal by space law already

**Comment by TermedClepe** [comment]
Define wiring to the grid, you can just as easily crit people with maxing all the SMES on station or building more.

**Comment by 9600bauds** [comment]
Dangerously excessive amounts of power on grid

**Comment by CptWad** [comment]
I can now imagine a mommi being put in jail for wireing an autistic-sized solars to the grid

**Comment by Killette2** [comment]
What if we added stupid fun stuff to do with power, like build tesla coils

**Comment by AutismSpider** [comment]
@TermedClepe  can upgrade the SMES so they all out put 400k into the net with T3 parts 300k with T2 parts.

so nothing is really safe if it gets upgraded it shit out ton of power into the net
